doc/rippled-example.cfg

```diff
@@ -987,23 +972,6 @@ pool.ntp.org
 [ips]
 r.ripple.com 51235
 
-# Public keys of the validators that this rippled instance trusts.  The latest
```

Comment:
This file itself is documentation, which is why @nbougalis and I are being so cautious.

> What would `[validators_file]` be set to?

That _is_ the question, isn't it. I looked again at the `Config` code and was reminded that it'll attempt a default file name if one isn't configured (https://github.com/wilsonianb/rippled/blob/val-file/src/ripple/core/impl/Config.cpp#L437-L467).

So what I'm thinking right now is that `[validators_file]` should be set to that default value, `validators.txt`. Why bother? Two reasons: (1) It indicates to someone reading through the sample file as part setting up a new instance that the are going to need a validators file (and maybe to go dig for more documenation). (2) It'll log an error if the file is missing. If we stick with the default, it gives up silently.